runtime/agent/CMakeLists.txt

```diff
@@ -20,5 +25,24 @@ target_link_libraries(bpftime-agent
   -ldl
   spdlog::spdlog
   bpftime_frida_uprobe_attach_impl
-  bpftime_syscall_trace_attach_impl
 )
+
+else()
+target_include_directories(bpftime-agent
+  PRIVATE
+  ${FRIDA_GUM_INSTALL_DIR}
+  ../include
+  ../../third_party/
+  ${SPDLOG_INCLUDE}
+  ${FRIDA_UPROBE_ATTACH_IMPL_INCLUDE}
+)
+target_link_libraries(bpftime-agent
+  ${FRIDA_GUM_INSTALL_DIR}/libfrida-gum.a
+  runtime
+  -lpthread
+  -lm
+  -ldl
+  spdlog::spdlog
+  bpftime_frida_uprobe_attach_impl
```

Comment:
Same as above, should `bpftime_frida_uprobe_attach_impl` be removed without bpf?